analysis/microbeMicrobeInteractions/tutorial_microbeMicrobeInteractions.m

```diff
@@ -13,21 +10,21 @@
 % tutorial can be adapted to any number of AGORA models and dietary conditions 
 % analyzed.
 % 
-% 
-% 
+initCobraToolbox(0);
+% path to where the COBRA Toolbox is located
+global CBTDIR
+cd([CBTDIR filesep 'tutorials' filesep 'analysis' filesep 'microbeMicrobeInteractions'])
```

Comment:
do you need the global here?
Wouldn't:
`fileparts(which('tutorial_microbeMicrobeInteractions.m'))` 
do what you want? It would have the added benefit of being independent of the actual folder structure, so if any time in the future, the structure changes again, this tutorial would not have to be modified and would still work as is.
The same actually applies for tutorial_hostMicrobeInteractions.m and testMgPipe

overall this would be something like:
```
tutorialPath = fileparts(which('tutorial_microbeMicrobeInteractions'));
cd(tutorialPath);
```
And `tutorialPath` could later be used to save and address other files, without always using CBTDIR and the whole pathes.

Reply:
Agree. @almut-heinken , may you change that? Thanks.